### ISSUE TITLE
hardhat-vyper fails to detect docker on windows

### ISSUE THREAD

**Issue by wighawag** [nomiclabs/hardhat#1364]
`hardhat-vyper` fails with this error : 

```
Error in plugin @nomiclabs/hardhat-vyper: Docker Desktop is not installed.
Please install it by following the instructions on https://www.docker.com/get-started

NomicLabsHardhatPluginError: Docker Desktop is not installed.
Please install it by following the instructions on https://www.docker.com/get-started
    at validateDockerIsInstalled (C:\dev\wighawag\template-ethereum-contracts\node_modules\@nomiclabs\hardhat-vyper\src\compilation.ts:175:11)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at compile (C:\dev\wighawag\template-ethereum-contracts\node_modules\@nomiclabs\hardhat-vyper\src\compilation.ts:38:3)
    at SimpleTaskDefinition.action (C:\dev\wighawag\template-ethereum-contracts\node_modules\@nomiclabs\hardhat-vyper\src\index.ts:25:3)
    at Environment._runTaskDefinition (C:\dev\wighawag\template-ethereum-contracts\node_modules\hardhat\src\internal\core\runtime-environment.ts:217:14)
    at Environment.run (C:\dev\wighawag\template-ethereum-contracts\node_modules\hardhat\src\internal\core\runtime-environment.ts:130:14)
    at SimpleTaskDefinition.action (C:\dev\wighawag\template-ethereum-contracts\node_modules\hardhat\src\builtin-tasks\compile.ts:1390:7)
    at Environment._runTaskDefinition (C:\dev\wighawag\template-ethereum-contracts\node_modules\hardhat\src\internal\core\runtime-environment.ts:217:14)
    at OverriddenTaskDefinition._action (C:\dev\wighawag\template-ethereum-contracts\node_modules\@typechain\hardhat\src\index.ts:28:5)
    at Environment._runTaskDefinition (C:\dev\wighawag\template-ethereum-contracts\node_modules\hardhat\src\internal\core\runtime-environment.ts:217:14)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Even though Docker Desktop is installed and running

**Comment by wighawag** [comment]
Looking deeper into hardhat code and it looks like hardhat-docker itself do not support windows.

Would be great to keep hardhat cross-platform at its core

**Comment by fvictorio** [comment]
Related: #1397 (I guess)

**Comment by alcuadrado** [comment]
I think we should drop docker and use the vyper binaries. They are released on github. That's what brownie uses, so they work correctly.

**Comment by feuGeneA** [comment]
Released in [@nomiclabs/hardhat-vyper v3.0.0](https://github.com/nomiclabs/hardhat/releases/tag/%40nomiclabs%2Fhardhat-vyper%403.0.0)
